src/main/java/gregtech/common/tileentities/machines/multi/MTEIntegratedOreFactory.java

```diff
@@ -203,7 +203,7 @@ protected MultiblockTooltipBuilder createTooltip() {
         tt.addMachineType("Ore Processor, IOF")
             .addInfo("Does all ore processing in one step")
             .addStaticParallelInfo(1024)
-            .addInfo("Every ore costs 30EU/t, 2L lubricant, 200L distilled water")
+            .addInfo("Every ore costs 30EU/t, extra costs 2L lubricant, 200L distilled water")
```

Comment:
This isn't really clear on what 'extra costs' means. Does this sound better?

```suggestion
            .addInfo("Every ore costs 30EU/t, 2L lubricant, 200L distilled water")
            .addInfo("Special recipes require their extra inputs on top of the normal costs")
```

Reply:
better than tooltips now. but player still doesn't know what is special recipes
maybe add another line that is "including chem bath recipes"?

Reply:
I think it's good now that it specifies "recipes with extra inputs."